### ISSUE TITLE
Again this problem: Conflict Between Helix Ultimate 2.0.6 And Popup With Jce Mediabox

### ISSUE THREAD

**Issue by gioacchino54** [JoomShaper/helix-ultimate#257]
Hi,
I changed the two files as an indicator in the solution: https://github.com/JoomShaper/helix-ultimate/commit/bfab89724112cd9a37c8e7d36323a41370576729
But the result hasn't changed the JCE Popup remains to not work.
Best Rgds
Gioacchino

Steps to reproduce the issue
[https://www.loom.com/share/e1c4a218976840968bc2640863f26fd7](https://github.com/JoomShaper/helix-ultimate/issues/url)

Expected result
[https://www.loom.com/share/e1c4a218976840968bc2640863f26fd7](https://github.com/JoomShaper/helix-ultimate/issues/url)

Actual result
System information (as much as possible)
Joomla! 3.10.3
Php 7.4.x
Helix Ultimate 2.06

Additional comments
I am configuring, on a local server, a pop through JCE Mediabox. It used to work, now with HU 2.0.6, it doesn't work anymore. To show what I mean I used the Protostar template. You can see the behavior in the video, alternating templates (Helix Ultimate and Protostar) you can see the wrong behavior.

**Comment by mdamjadrana** [comment]
we tested it works. can you post in [forum](https://www.joomshaper.com/forum)  with your live link. so that we can check.

**Comment by gioacchino54** [comment]
> we tested it works. can you post in [forum](https://www.joomshaper.com/forum) with your live link. so that we can check.

https://www.joomshaper.com/forum/question/11666

**Comment by ittoufiq** [comment]
Did you check the latest version of Helix Ultimate v2.0.9? Please let us know. Thanks

**Comment by gioacchino54** [comment]
Yes of course, the problem occurs after upgrading from version 2.0.8 to version 2.0.9.
With version 2.0.8 it worked, but after upgrading to version 2.0.9 it started to no longer work. As seen in the video: https://www.loom.com/share/d26b3c3d04f0471497f507fd6ea6cc6a
If you see as soon as I upgrade to 2.0.9 it stops working. 
Best Rgds

**Comment by gioacchino54** [comment]
I upgraded to 2.0.10 and the problem persists

**Comment by joomleb** [comment]
@gioacchino54 
Please, Can you confirm it with last Joomla 4.4.5 + Helix Ultimate 2.1.1 ?

Which version of PHP are you running?

**Comment by gioacchino54** [comment]
> @gioacchino54 Please, Can you confirm it with last Joomla 4.4.5 + Helix Ultimate 2.1.1 ?
> 
> Which version of PHP are you running?

There are still some problems, which I detected on the Google Chrome Browser, while on Firefox it is not detected, as you can see in this video https://www.loom.com/share/515b24eb37f64262b47e739cc16806d9?sid=b042b5e0-e41d-4e38-9a51 -639ea1f5b67c
While the initial problem I reported seems to have been resolved.

In the test I did I used:
php 8.1
Joomla! 4.4.5
Joomla! 5.1.0

Best Rgds
